src/main/java/frc/robot/commands/VisionDriverAid.java

```diff
@@ -25,19 +31,26 @@
   Supplier<Double> m_forwardVal;
   Supplier<Double> m_rotateVal;
   PIDController m_pid;
+  Command shooter;
+  Command feed;
+  Command shooterAuto;
 
   public VisionDriverAid(Joystick driveJoystick, Supplier<Double> rotateVal) {
     m_forwardVal = ()-> sign(Config.removeJoystickDeadband(driveJoystick.getRawAxis(Config.LEFT_CONTROL_STICK_Y)), Config.INVERT_FIRST_AXIS);
     // Use addRequirements() here to declare subsystem dependencies.
     addRequirements(DriveBaseHolder.getInstance());
     m_rotateVal = rotateVal;
     m_pid = new PIDController(0, 0, 0);
+    shooter = new SpinUpShooterWithTime(2000, 0);
```

Comment:
2000 RPM is the low goal RPM around the vicinity of the tarmac. We can tune this value to one specific spot or we can use auto shooting.

Reply:
2000 RPM is set at the beggining of the driver aid to spin up the shooter. Once the driver aid knows it's aligned, it schedules David's vision distance based RPM.